tests/unit/test_views.py

```diff
@@ -628,13 +628,12 @@ def test_security_key_giveaway_not_found(self):
         with pytest.raises(HTTPNotFound):
             SecurityKeyGiveaway(request).security_key_giveaway()
 
-    def test_security_key_giveaway(self):
+    def test_security_key_giveaway(self, monkeypatch):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏is `monkeypatch` needed here? I don't think it's being used, is it?

Reply:
You’re right, `monkeypatch` is not being used here. Must have been something else I was trying before I worked out this fix.